Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -204,6 +204,26 @@ def ap_find_tests(bld, use=[]):
             cxxflags=['-Wno-undef'],
         )
 
+_versions = []
+
+@conf
+def ap_version_append(ctx, k, v):
+    ctx.env['AP_VERSION_ITEMS'] += [(k, os.environ.get(k, v))]
+
+@conf
+def ap_version_append_str(ctx, k, v):
+    ctx.env['AP_VERSION_ITEMS'] += [(k, '"{}"'.format(os.environ.get(k, v)))]
```

Comment:
Ok. As you know we usually don't have unused code, but since this is in the build system I don't think it matters. I'm totally fine with it.